dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -891,6 +895,12 @@
             "modelAsString": true
           }
         },
+        "confidenceScore": {
+          "x-ms-client-name": "confidence",
```

Comment:
Per offline discussion, seems `confidenceScore` is already used in many places so I'd remain consistent. /cc @KrzysztofCwalina @tg-msft @jsquire about the inconsistency here with recommendations for other Cognitive Services - Language pillar SDKs. Would you all agree it's better to be consistent for the TA SDK than other Language SDKs?

Reply:
not sure i get that, language SDK is replacing TA SDK long run. ConfidenceScore should be the preferred term anyway, confidence does not convey a number (e..g score)

Reply:
There's no actual "Language SDK". We still have separate TA, CLU, QA, etc. SDKs all as part of the Language unification, but - for various reasons (like size, focus, etc.) we still ship separate SDKs.

So, because TA was and still is a separate SDK, it makes more sense to continue using `ConfidenceScore` as a property instead of just `Confidence` that other, separate SDKs are using.

Reply:
Keeping confidenceScore as the field